test/potato_udp_test.erl

```diff
@@ -0,0 +1,12 @@
+-module(potato_udp_test).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+potato_udp_test() ->
+  Port = 3142,
+  {ok, Pid} = gen_server:start(potato_udp, Port, []),
+  %% ?debugFmt("started potato_udp gen_server ~p~n",[Pid]),
+  gen_server:cast(Pid, {send, "localhost", Port, "hi"}),
```

Comment:
you forgot to pack before unpacking??

your unpack throws badarg exception see
https://travis-ci.org/antonbobkov/potato_game/builds/442478411?utm_source=github_status&utm_medium=notification

```
got: <<"hi">>
terminate reason: {badarg,
                      [{erlang,binary_to_term,[<<"hi">>],[]},
                       {messages,unpack,1,
```

Reply:
should be `messages:pack("hi")` maybe?

or do it inside `handle_cast`

Reply:
🙀

Reply:
it's passing on my computer 😱